std_msgs/Header header
myAprilTagDetection[] detections
